src/Aspire.Cli/Program.cs

```diff
@@ -270,6 +272,66 @@ private static void ConfigureNewCommand(Command parentCommand)
         parentCommand.Subcommands.Add(command);
     }
 
+    private static Integration GetIntegrationByInteractiveFlow(IEnumerable<Integration> knownIntegrations)
+    {
+        var prompt = new SelectionPrompt<Integration>()
+            .Title("Please select the integration you want to add:")
```

Comment:
how does this work when you need to add the hosting package as well?

Reply:
We will actually only be adding the hosting packages.